--- llvm-2.8/lib/ExecutionEngine/JIT/Intercept.cpp	2010-08-31 03:24:03.000000000 +0530
+++ llvm-2.8-new/lib/ExecutionEngine/JIT/Intercept.cpp	2019-04-30 16:12:40.000000000 +0530
@@ -19,6 +19,14 @@
 #include "llvm/Support/ErrorHandling.h"
 #include "llvm/System/DynamicLibrary.h"
 #include "llvm/Config/config.h"
+
+#if defined(__linux__)
+#if defined(HAVE_SYS_STAT_H)
+#include <sys/stat.h>
+#include <unistd.h>
+#endif
+#endif
+
 using namespace llvm;
 
 // AtExitHandlers - List of functions to call when the program exits,
